packages/babel-helpers/src/helpers/applyDecs.js

```diff
@@ -491,13 +491,8 @@ function applyMemberDecs(
     );
   }
 
-  if (protoInitializers.length > 0) {
-    pushInitializers(ret, protoInitializers);
-  }
-
-  if (staticInitializers.length > 0) {
-    pushInitializers(ret, staticInitializers);
-  }
+  pushInitializers(ret, protoInitializers);
+  pushInitializers(ret, staticInitializers);
```

Comment:
A dub initializer should be pushed when the member decorators do not introduce initializers, which only known at runtime.